pkg/obfuscator/common.go

```diff
@@ -0,0 +1,28 @@
+package obfuscator
+
+import (
+	"fmt"
+
+	"k8s.io/klog/v2"
+)
+
+// obfsGenerator consists of the required fields for the consistent,static obfuscations and the count of the obfuscations
+// This implements the methods static, consistent inorder to return the required replacement based on the replacementType
+type obfsGenerator struct {
```

Comment:
I'd put that into a generator.go file instead

Reply:
and I guess you can also name it that way, package is already called obfuscator

Reply:
Sure.